recipes/r-rcppplanc/meta.yaml

```
@@ -0,0 +1,118 @@
+{% set version = '1.0.0-rc2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rcppplanc
+  version: {{ version|replace("-", "_") }}
+
+
+source:
+  url: https://github.com/welch-lab/RcppPlanc/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 20663306ba3f52073a33d435e14015d3667f3b266d820379364b53cce0606f20
+
+build:
+  skip: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, withr, rmarkdown, testthat (>= 3.0.0)
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+    - {{ stdlib("c") }}
+    - cmake >=3.22
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - pkg-config
+    - diffutils  # [not win]
+    - m2-diffutils  # [win]
+    - patch  # [not win]
+    - m2-patch  # [win]
+
+  host:
+    - r-base
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppprogress
+    - libhwloc >=2.0.0,<3.0.0
+    - hdf5 >=1.10
+    - libcblas
+    - liblapack
+
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppprogress
+    - libblas  # [not (osx and arm64)]
+    - libblas * *accelerate  # [osx and arm64]
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('RcppPlanc')"           # [not win]
+    - "\"%R%\" -e \"library('RcppPlanc')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://github.com/welch-lab/rcppplanc
+  license: GPL-2.0-or-later
+  summary: '''Rcpp'' bindings for ''PLANC'', a highly parallel and extensible NMF/NTF  (Non-negative
+    Matrix/Tensor Factorization) library. Wraps algorithms described in Kannan et. al
+    (2018) <doi:10.1109/TKDE.2017.2767592> and Eswar et. al (2021) <doi:10.1145/3432185>.
+    Implements algorithms described in Welch et al. (2019) <doi:10.1016/j.cell.2019.05.006>,
+    Gao et al. (2021) <doi:10.1038/s41587-021-00867-x>, and Kriebel & Welch (2022) <doi:10.1038/s41467-022-28431-4>.'
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - theAeon
```

Conda Forge R team would like to be listed:

```suggestion
    - conda-forge/r
    - theAeon
```